api/transport/handler.go

```diff
@@ -65,6 +69,9 @@ func (h HandlerSpec) Unary() UnaryHandler { return h.unaryHandler }
 // Oneway returns the Oneway Handler or nil
 func (h HandlerSpec) Oneway() OnewayHandler { return h.onewayHandler }
 
+// Stream returns the Stream Handler or nil
```

Comment:
nit: Most of the comments in this file or above don end with a period. maybe we want to fix that?